json-rpc/src/views.rs

```diff
@@ -40,3 +40,52 @@ impl From<&[u8]> for BytesView {
         Self(hex::encode(bytes))
     }
 }
+
+#[derive(Serialize)]
+pub struct TransactionView {
+    pub version: u64,
+    pub transaction: TransactionData,
+}
+
+#[derive(Serialize)]
+pub struct TransactionData {
```

Comment:
nit: can we use consistent naming (e.g. View suffix)

Reply:
fixed